extension/src/experiments/columns/collect/timestamp.ts

```diff
@@ -0,0 +1,14 @@
+import { ColumnAccumulator } from './util'
+import { ColumnType } from '../../webview/contract'
+
+export const timestampColumn = {
```

Comment:
Is there a better place to keep this variable? It is used in a couple different files 🤔 

<img width="612" alt="image" src="https://user-images.githubusercontent.com/43496356/186456185-a8989cb2-4e19-463a-a57d-2ad010c6f6d8.png">

Reply:
TBH I would make a `extension/src/experiments/columns/constants.ts` file. I would also not collect this column for every iteration in `extension/src/experiments/columns/collect.ts`. I would create the initial accumulator object with this column in it.

Importing into the test fixtures breaks the `turborepo` workflow (a change in this file will not necessarily trigger a rerun of the webview tests but they do depend on this).

Reply:
You should be able to update the dependencies for turborepo,

Reply:
> You should be able to update the dependencies for turborepo.

Makes sense! I've created a constants file but I'm not quite sure how to update the dependencies for turborepo 🤔

Reply:
There is a `globalDependencies` key in `turbo.json`. Docs: https://turborepo.org/docs/reference/configuration#globaldependencies

Reply:
Thanks! Updated!